tests/test_router_bgp.rb

```diff
@@ -399,12 +400,38 @@ def test_default_cluster_id
   def test_enforce_first_as
     asnum = 55
     bgp = RouterBgp.new(asnum)
-    bgp.enforce_first_as = true
-    assert(bgp.enforce_first_as,
-           'bgp enforce-first-as should be enabled')
-    bgp.enforce_first_as = false
-    refute(bgp.enforce_first_as,
-           'bgp enforce-first-as should be disabled')
+    if platform == :ios_xr
+      @device.cmd('term width 300')
+      # Lets assert based on XR standard output if a command is not found
+      pattern = 'No such configuration item'
+
+      # Test default
+      s = @device.cmd("show running-config router bgp #{asnum} " \
+                      'bgp enforce-first-as disable')
+      assert_match(pattern, s,
```

Comment:
Good idea. I am not sure I gain much though for the number of asserts I am doing. Will take a look though.